tests/unit/components/WalletAssetDetails.spec.ts

```diff
@@ -1,52 +1,48 @@
-// import Vuex from 'vuex'
-// import { shallowMount, createLocalVue } from '@vue/test-utils'
-
-// import WalletAssetDetails from '@/components/WalletAssetDetails.vue'
-// import { TranslationMock, SoramitsuElementsImport } from '../../utils'
-
-// TODO: Fix CI
-
-// const localVue = createLocalVue()
-// localVue.use(Vuex)
-// const store = new Vuex.Store({
-//   modules: {
-//     Account: {
-//       getters: {
-//         selectedAssetDetails: () => ([
-//           {
-//             operation: 'SWAP',
-//             fromAmount: 100,
-//             toSymbol: 'KSM',
-//             toAmount: 24390.1239,
-//             status: 'IN_PROGRESS',
-//             date: 1605048643745
-//           }
-//         ])
-//       },
-//       actions: {
-//         getAssetDetails: jest.fn()
-//       }
-//     },
-//     Router: {
-//       getters: {
-//         currentRouteParams: () => ({ symbol: 'XOR' })
-//       },
-//       actions: {
-//         navigate: jest.fn()
-//       }
-//     }
-//   } as any
-// })
-
-describe('WalletAssetDetails.vue', () => {
-  beforeEach(() => {
-    // SoramitsuElementsImport(localVue)
-    // TranslationMock(WalletAssetDetails)
+import Vuex from 'vuex';
+import { shallowMount } from '@vue/test-utils';
+import { useDescribe, localVue } from '../../utils';
+
+import WalletAssetDetails from '@/components/WalletAssetDetails.vue';
+import { MOCK_ACCOUNT_ASSETS, MOCK_FIAT_PRICE_AND_APY_OBJECT, MOCK_HISTORY } from '../../utils/mock';
+import { MOCK_ACCOUNT } from '../../utils/WalletAccountMock';
+
+const createStore = (isNotXor = false) =>
+  new Vuex.Store({
+    getters: {
+      accountAssets: () => MOCK_ACCOUNT_ASSETS,
+      currentRouteParams: () => ({ asset: MOCK_ACCOUNT_ASSETS[Number(isNotXor) | 0] }),
```

Comment:
`Number(isNotXor) | 0` For what purpose here is used Bitwise OR operator? Seems like a convertion to a number is enough `Number(isNotXor) `